hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdiff/RocksDBCheckpointDiffer.java

```diff
@@ -1386,14 +1205,9 @@ public boolean shouldRun() {
     return !suspended.get();
   }
 
-  @VisibleForTesting
-  public boolean debugEnabled(Integer level) {
-    return DEBUG_LEVEL.contains(level);
-  }
-
   @VisibleForTesting
   public ConcurrentHashMap<String, CompactionNode> getCompactionNodeMap() {
-    return compactionNodeMap;
+    return (ConcurrentHashMap<String, CompactionNode>) compactionDag.getCompactionMap();
```

Comment:
It is Ok to return like this in this patch, but we should do a clean-up and use  `CompactionDag` as encapsulation.

Reply:
IMO this method should be rewritten as ConcurrentMap.
In general, returning more abstract type is preferred, as it avoid a lot of unnecessary code change later, just like this.